CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fixed
 * Fixed an issue that would cause `Logger.Default` on Unity to always revert to `Debug.Log`, even when a custom logger was set. (Issue [#2481](https://github.com/realm/realm-dotnet/issues/2481))
 * Fixed an issue where `Logger.Console` on Unity would still use `Console.WriteLine` instead of `Debug.Log`. (Issue [#2481](https://github.com/realm/realm-dotnet/issues/2481))
+* [Unity] Preserved additional constructors necessary to serialize and deserialize Custom User Data. (PR [#2519](https://github.com/realm/realm-dotnet/pull/2519))
```

Comment:
The read me entry leads to this PR. Do we prefer to link to the PR or the issue itself (like it was done in the lines above)?

Reply:
As written in the PR itself, unfortunately no issue was generated for that. I don't know if I should have done that. It just encountered the issue and opened a PR right away to fix it. Maybe this isn't the way it's done, I'm not sure.